snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: errbot
+version: master
+summary: chatbot that brings your tools into the conversation.
+description: |
+  Errbot is a chatbot. It allows you to start scripts interactively from your
+  chatrooms for any reason: random humour, chatops, starting a build, monitoring
+  commits, triggering alerts...
+
+grade: devel
+confinement: strict
+
+apps:
+  errbot:
+    command: env LC_ALL=C.UTF-8 errbot
+    plugs: [home, network]
+
+parts:
+  errbot:
+    source: .
+    plugin: python
+    build-packages: [libffi-dev, libssl-dev]
+    stage-packages: [git]
```

Comment:
I think we need to add the build-packages to stage-packages as well as they're needed by pip when installing plugins with extra dependencies. We'll probably also have to add some additional common ones (`libxslt-dev` is another common one which comes to mind.

We should probably add mysql (`libmysqlclient-dev`) and psql (not sure which package provides it) libraries as well so people can install python packages needing to link to those, such as when using our sql storage plugin.